kotlin-analysis-api/src/test/kotlin/com/google/devtools/ksp/processor/HelloWorldProcessor.kt

```diff
@@ -0,0 +1,43 @@
+package com.google.devtools.ksp.processor
+
+import com.google.devtools.ksp.processing.Resolver
+import com.google.devtools.ksp.symbol.KSAnnotated
+
+class HelloWorldProcessor : AbstractTestProcessor() {
+    val results = mutableListOf<String>()
+
+    override fun process(resolver: Resolver): List<KSAnnotated> {
+        val fooKt = resolver.getAllFiles().single()
+        results.add(fooKt.fileName)
+
+        // FIXME: symbols escape contexts ¯\_(ツ)_/¯
```

Comment:
good catch, after taking a look into it, it needs some rework on how KSP symbol implementations wraps the `KtSymbol`s